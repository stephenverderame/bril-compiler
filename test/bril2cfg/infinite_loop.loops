digraph {
	subgraph cluster_main {
		label="main()";
		rankdir="TB";
		main_0 [label="START", shape="rectangle", style="rounded"];
		subgraph cluster_main_loop0 {
			label="";
			color="blue";
			style="rounded";
			bgcolor="#FFFFFF00";
			main_0;
		}

		main_0 -> main_0;
	}
}
