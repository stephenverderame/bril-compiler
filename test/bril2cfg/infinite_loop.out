digraph {
	subgraph cluster_main {
		label="main";
		rankdir="TB";
		main_0 [label="START", shape="ellipse"];

		main_0 -> main_0;
	}
}
